sbt-metals/src/sbt-test/sbt-metals/setup/test

```diff
@@ -0,0 +1,9 @@
+> semanticdbEnable
+> compile
+$ exists target/scala-2.12/classes/META-INF/semanticdb/A.semanticdb
+
+$ absent .metals/buildinfo/
+> Compile/metalsWriteBuildInfo
```

Comment:
Should we test `metalsSetup` instead? Unless users are expected to run `Compile/metalsWriteBuildInfo`, I think it's better to not test implementation details like this.

Reply:
out of curiosity, `Compile/metalsWriteBuildInfo` won't work on 0.13. How hard would it be add 0.13 scripted tests?